launcher/commands/usvfs.py

```diff
@@ -34,8 +34,16 @@ def run(self, args) -> None:
         gamma_dir = Path(args.gamma).expanduser()
         anomaly_dir = Path(args.anomaly).expanduser()
         install_dir = Path(args.final).expanduser()
-
-        install_dir.mkdir(parents=True)
+        if install_dir.exists():
```

Comment:
Why not doing this instead ?
* Reading *user.ltx*
* Deleting install_dir (with rmtree)
* mkdir()